src/App/App.js

```diff
@@ -0,0 +1,29 @@
+import './App.scss';
+import Header from '../Header';
+import Card from '../Card';
+
+
+function App() {
+  const loremIpsum = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras vestibulum mauris justo, non egestas sapien malesuada sit amet.';
+
+  const cards = [0, 1, 2].map(one =>
+    <Card
+      key={one}
+      caption={'Card header'}
+      content={loremIpsum}
+    />
+  );
```

Comment:
Can you please move array of cards to variable? Also objects in this array should have identity field using for key value in reducing and it will be helpful in future.

Reply:
[0, 1, 2] are already works as ids.
Correct me if I'm understand you in a wrong way, please.

Reply:
The Idea is to move array to separated variable contains objects. Can you please imagine this array comes from BE?
Like
`
[
    {
        id: 'id',
        caption: 'caption',
        content: 'content',
    },
    ...
]
`

Reply:
Got you!